maingate/src/range.rs

```diff
@@ -519,8 +532,9 @@ mod tests {
         let inputs = (2..20)
             .map(|number_of_limbs| {
                 let bit_len = LIMB_BIT_LEN * number_of_limbs + OVERFLOW_BIT_LEN;
+                let value = BigUint::from(1u32) << 16 - 1;
                 Input {
-                    value: Value::known(Fp::from_u128((1 << bit_len) - 1)),
+                    value: Value::known(Fp::from_raw(split_biguint(value))),
```

Comment:
I think you can just do something like `Fp::from_repr(value.to_bytes_le()).into()` to convert a `BigUint` to a field element.